packages/extensions/openapi-to-typespec/src/main.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
The license file should be added into all source files, but you could do that later.